l1-contracts/contracts/bridge/L1AssetRouter.sol

```diff
@@ -538,449 +423,38 @@ contract L1AssetRouter is IL1AssetRouter, ReentrancyGuard, Ownable2StepUpgradeab
         uint16 _l2TxNumberInBatch,
         bytes32[] calldata _merkleProof
     ) public nonReentrant whenNotPaused {
```

Comment:
This function, can you have it be called from the Nullifier please, and only from the nullifer?